examples/alerts/alerts.yaml

```diff
@@ -301,16 +301,6 @@ groups:
       severity: warning
 - name: thanos-sidecar
   rules:
-  - alert: ThanosSidecarPrometheusDown
```

Comment:
Is this intentional?

Reply:
Description or Title of this PR is not mentioning this change 🤔

Reply:
@bwplotka Yes it was intensional, but excluding it in the title is not :) 

I've removed it as it would fire at the same time as ThanosSidecarPrometheusDown., so found it to be redundant. Let me amend the title and description to add info above removal of ThanosSidecarPrometheusDown.

More details: https://github.com/thanos-io/thanos/issues/3915#issuecomment-885475143

Reply:
`ThanosSidecarPrometheusDown` will be the same as `ThanosSidecarPrometheusDown`? 🤔 Typo?

Reply:
ping 
![image](https://user-images.githubusercontent.com/6950331/129752273-953e14d3-e54b-430c-90bc-d6131a579405.png)

Reply:
@bwplotka PR description already has info about removal of `ThanosSidecarPrometheusDown`.